giraph-core/src/main/java/org/apache/giraph/utils/UnsafeByteArrayOutputStream.java

```diff
@@ -197,6 +201,13 @@ public void writeChar(int v) throws IOException {
     position += SIZE_OF_CHAR;
   }
 
+  @Override
+  public void writeChar(char v) {
```

Comment:
Just making sure, there are no other methods that are overloaded other than this one?